Gemfile

```diff
@@ -40,3 +40,6 @@ group :development, :test do
   gem 'teaspoon-jasmine'
   gem 'selenium-webdriver'
 end
+
+
+gem "mumuki-domain", github: "mumuki/mumuki-domain", branch: "chore-database-optimization"
```

Comment:
Please remove and rebase when ready to merge. I didn't want to rebase it right now just to show that each optimization commit is green :green_circle: